components/PhoneTrackingButton.vue

```diff
@@ -29,42 +32,53 @@ export default defineComponent({
   },
   methods: {
     startTracking () {
-        this.watcherId = navigator.geolocation.watchPosition((result) => {
-          if (!this.tracking) {
-            this.$bvToast.toast("You are being tracked.", {
-              title: "Paradar message",
-              autoHideDelay: 3000,
-              variant: "danger",
-              solid: true,
-              toaster: "b-toaster-top-center"
-            });
-            this.tracking = true;
-          }
-          this.$handleUpdatedPosition(this.client, result);
-        });
-    },
-    stopTracking () {
-        this.tracking = false;
-        navigator.geolocation.clearWatch(this.watcherId);
-        this.watcherId = -1;
-        this.$bvToast.toast("You are not being tracked anymore.", {
+      this.watcherId = navigator.geolocation.watchPosition((result) => {
+        if (!this.tracking) {
+          this.$bvToast.toast("You are being tracked.", {
             title: "Paradar message",
-            autoHideDelay: 3000,
-            variant: "success",
+            autoHideDelay: 200000,
```

Comment:
Why show the toast for 3.3 minutes? It remains even after clicking the 'stop tracking' button.

Reply:
Sorry forgot to put it back. Done now

Reply:
👍